### PR TITLE
Fix issue with hover animations

### LINKED ISSUE #235: [FEATURE]: Adding more hover animations
### Feature Description 📝

Hover animation section looks empty and requires some love, if you have some nice hover animations in mind, drop that idea below so that it can be added to SyntaxUI 

### Screenshots 📸

CleanShot 2024-06-19 at 18 31 55@2x

### PR DESCRIPTION
**Pull request by Asin-Junior-Honore**
## Description

This pull request introduces additional hover animations to enhance user interaction across the application. Specifically, it adds the Pulse animation to the HoverPulseButton component.

## Related Issue

Fixes #235 

## Proposed Changes

Implemented Pulse animation in HoverPulseButton component.
Added necessary CSS and keyframes for the Pulse animation.
Ensured compatibility and smooth functionality across browsers.

## Screenshots

Screenshot (24)

## Checklist

Please check the boxes that apply:

- [x] I have rebased my branch on top of the latest `main` branch.
- [x] I have tested the changes locally
- [x] I ran `npm run build` and build is successful
- [ ] I have added necessary documentation (if applicable)
- [ ] I have updated the credits.md file (if applicable)

## Additional Context

Please provide any additional context or information about the pull request here.

### CODE REVIEW

**Comment by epoll31** [comment]
Could you swap out the pure CSS classes for inline TailwindCSS classes?

**Comment by Asin-Junior-Honore** [comment]
Hey 
@epoll31 ,

I have swapped out the pure CSS classes for inline TailwindCSS classes as requested. Please review my PR at your earliest convenience.

Thanks!

File: src/styles/tailwind.css [hunk 1]
```diff
@@ -25,22 +25,3 @@
...
-  syntax: "<angle>";
-  inherits: false;
-  initial-value: 0deg;
-}
-
-@property --discord-button-x {
-  syntax: "<length>";
-  inherits: false;
-  initial-value: 0;
-}
```
Comment (epoll31) [inline src/styles/tailwind.css:46]: Can you restore the changes in this file? I believe that it can remain the same as it was before the PR.

**Review by epoll31 — COMMENTED** [review]
I am just looking for general consistency issues now. I left a comment on the `tailwind.css` file. But, as for opinionated styling decisions and whether or not we should add this component to the showcase we should wait for @Ansub's opinions.

**Comment by Asin-Junior-Honore** [comment]
@epoll31 
I have just restored the content in the tailwind.css file , sorry about that please thanks😊🙇

File: src/app/(docs)/animations/hovers/pulse/page.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import PreviewCodeGroup from '@/components/code/PreviewCodeGroup'
+
+<Link className="flex flex-row items-center" href="/animations/hovers">
+  <ChevronLeft size={17} />
+  Back
+</Link>
+
+# Pulse Animation Example
+
+This example demonstrates a button with a pulse animation effect.
```
Comment (Ansub) [inline src/app/(docs)/animations/hovers/pulse/page.mdx:13]: remove this description

File: tailwind.config.ts [hunk 3]
```diff
@@ -237,6 +241,7 @@ export default {
         'accordion-up': 'accordion-up 0.2s ease-out',
         'skew-scroll': 'skew-scroll 20s linear infinite',
         buttonheartbeat: 'buttonheartbeat 2s infinite ease-in-out',
+        pulse: 'pulse 1s',
```
Comment (Ansub) [inline tailwind.config.ts:244]: name it `hover-pulse`

File: tailwind.config.ts [hunk 4]
```diff
@@ -85,6 +85,10 @@ export default {
             transform: 'scale(1)',
           },
         },
+        pulse: {
```
Comment (Ansub) [inline tailwind.config.ts:88]: name it same like other hover animations and keyframes

File: src/styles/tailwind.css [hunk 5]
```diff
@@ -34,13 +34,13 @@
   font-size: 14px;
 }
 @property --discord-button-angle {
-  syntax: "<angle>";
```
Comment (Ansub) [inline src/styles/tailwind.css:37]: since there are no changes in this file can you revert this file back to how it is on production

**Comment by Asin-Junior-Honore** [comment]
Hey
 @Ansub ,

The requested changes and adjustments have been made. Please review at your earliest convenience.

File: tailwind.config.ts (hunk 4 above)
Comment (Asin-Junior-Honore) [reply tailwind.config.ts]: Done

File: src/styles/tailwind.css (hunk 5 above)
Comment (Asin-Junior-Honore) [reply src/styles/tailwind.css]: Done aswell sir👍

File: src/app/(docs)/animations/hovers/pulse/page.mdx [hunk 6]
```diff
@@ -0,0 +1,37 @@
+import Link from 'next/link'
```
Comment (Ansub) [inline src/app/(docs)/animations/hovers/pulse/page.mdx:1]: can you rebase and add the meta data here like it's added to other hover animations?

**Comment by Ansub** [comment]
Looks good, i just added one comment, you just have to copy and past meta data from other hover animations and paste it in your pulse animation and simply change the copy from `so-and-so` hover animation to `pulse`. 

once done we can get this merged. Thanks for your contribution!

File: src/app/(docs)/animations/hovers/pulse/page.mdx (hunk 6 above)
Comment (Asin-Junior-Honore) [reply src/app/(docs)/animations/hovers/pulse/page.mdx:1]: ok i have rebased but i dont understand the meta part you talked about ,please maybe you could give more context please 🙏

Comment (Ansub) [reply src/app/(docs)/animations/hovers/pulse/page.mdx:1]: add this on top of the file: 

```
export const metadata = {
  title: 'Pulse Hover Effect',
  description: 'A Pulse hover effect animation that gives a playful and energetic effect to your React and Next.js projects, built using Tailwind CSS.',
} 

```

Comment (Asin-Junior-Honore) [reply src/app/(docs)/animations/hovers/pulse/page.mdx:1]: ooh thank you very much 🙏😊 i see,let me do that and push asap

**Comment by Asin-Junior-Honore** [comment]
hey 
@Ansub 
I have added the meta data for the hover-pulse animation as requested. Please review at your earliest convenience.

**Review by Ansub — APPROVED** [review]

**Comment by Ansub** [comment]
Thank you for your valuable contribution to SyntaxUI! 🌟

Are you part of our [SyntaxUI Discord](https://syntaxui.com/discord) community yet? 
   • If yes, please reach out to me there – a special contributor badge awaits you!
   • If not, we'd love to have you join us. It's the perfect place to connect, share ideas, and stay updated on all things SyntaxUI.

Once again, thank you for helping shape the future of SyntaxUI. Your contribution makes a real difference! 💪✨

**Pull request merged**